traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -1645,6 +1645,16 @@ def get_server_delivery_services(self, server_id=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	@restapi.api_request('get', 'servers/details?hostName={name}', ('3.0','4.0',))
+    def get_server_details(self, name=None):
+        """
+		Retrieve server info in detail /api/4.0/servers/details?hostName=:name
+		:ref: `to-api-servers-details-hostName`
```

Comment:
This should be a reStructuredText reference to a a label in the documentation somewhere.

Example:

https://github.com/apache/trafficcontrol/blob/038a439ebedd57ec6a4df1b2e65a390e04d6a00a/traffic_control/clients/python/trafficops/tosession.py#L270-L273

corresponds with

https://github.com/apache/trafficcontrol/blob/038a439ebedd57ec6a4df1b2e65a390e04d6a00a/docs/source/api/v4/asns.rst#L17-L21